fixtures/chauvet-dj/slimpar-t12-usb.json

```diff
@@ -0,0 +1,198 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "SlimPAR T12 USB",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Anonymous"],
+    "createDate": "2021-08-26",
+    "lastModifyDate": "2021-08-26"
+  },
+  "comment": "8-CH Mode\nIf CH6 is 224-255 then Channel 5 Parameters are:\n000-010 -> No Function\n011-255 -> Sound Sensitivity Level",
+  "links": {
+    "manual": [
+      "https://www.chauvetdj.com/wp-content/uploads/2015/12/SlimPAR_T12_USB_UM_Rev1_WO.pdf"
+    ],
+    "productPage": [
+      "https://www.chauvetdj.com/products/slimpar-t12-usb/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=Qav_XTa3I68"
+    ]
+  },
+  "physical": {
+    "dimensions": [227, 84, 252],
+    "weight": 1.8,
+    "power": 39,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "12 LEDs (tri-color RGB) 3 W (1 A)",
+      "lumens": 817
+    },
+    "lens": {
+      "degreesMinMax": [15, 15]
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "ColorPreset",
+          "comment": "Color Macros"
+        }
+      ]
+    },
+    "Strobe/Speed": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction",
+          "comment": "When CH6 is 000-031"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "Speed",
+          "speedStart": "slow",
+          "speedEnd": "fast",
```

Comment:
The spec sheet says the strobe rate is 0-20 Hz, so this could be more specific, if desired.